packages/plugins/plugin-assistant/src/stories/Chat.stories.tsx

```diff
@@ -198,7 +202,7 @@ export const WithDocument = {
   }),
   args: {
     components: [ChatContainer, [SurfaceContainer, CommentsContainer, LoggingContainer]],
-    blueprints: ['dxos.org/blueprint/assistant'],
+    blueprints: [BLUEPRINT_KEY],
```

Comment:
better than a raw string, but still doesn't feel right. since we have a lot of keys (e.g., the translation key, plugin key, etc.) not sure what to do.